sui/gmp.js

```diff
@@ -18,8 +18,14 @@ const {
     utils: { arrayify },
 } = ethers;
 
-async function sendCommand(keypair, client, chain, args, options) {
+async function sendCommand(keypair, client, chains, args, options) {
     const [destinationChain, destinationAddress, feeAmount, payload] = args;
+
+    if (options.env !== 'local' && !chains[destinationChain]) {
+        throw new Error(`Chain ${destinationChain} not found in the config`);
```

Comment:
we shouldn't fail. it might be a partner chain that's not listed in our config, just print a warning that it's not found in the local config so it might not be a registered chain